app/models/user.rb

```diff
@@ -115,6 +116,26 @@ def hybrid?
     school_id && responsible_body_id
   end
 
+  # Wrapper methods to ease the transition from 'user belongs_to school',
+  # to 'user has_many schools'
+  def school
+    schools.first
+  end
+
+  def school_id
+    school&.id
+  end
+
+  def school_id=(new_school_id)
+    user_schools.delete_all
+    schools << School.find(new_school_id) if new_school_id
+  end
+
+  def school=(new_school)
+    user_schools.delete_all
+    schools << new_school if new_school.present?
```

Comment:
i feel this guard should also guard the line above? otherwise a mistake might delete all the associations

Reply:
Likewise, this supports the case of removing a user from a school by doing `user.update!(school: nil)` etc